auslib/admin/views/forms.py

```diff
@@ -144,6 +144,7 @@ class ScheduledChangeForm(Form):
     telemetry_channel = NullableStringField("Telemetry Channel")
     telemetry_uptake = NullableStringField("Telemetry Uptake")
     when = IntegerField("When", validators=[Optional(), not_in_the_past()])
+    change_type = StringField("Change Type")
```

Comment:
Also, did you try using a SelectField for this? There's only a few valid values, so it would be good to enforce that. See "update_type" for an example.